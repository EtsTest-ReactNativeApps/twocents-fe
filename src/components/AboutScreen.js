import React, { Component } from 'react';
import { Text, View, StyleSheet, Image } from 'react-native';

export class AboutScreen extends Component {
  static navigationOptions = ({ navigation }) => {
    return {
      title: 'About',
      headerStyle: {
        backgroundColor: 'white',
      },
    };
  };

  render() {
    return (
      <View style={styles.container}>
        <View style={styles.main}>
          <View style={styles.imageBox}>
            <Image
              source={require("../images/twocents-logo.png")}
              style={{ width: 200, height: 200 }}
            />
          </View>
          <Text style={styles.text}>TwoCents</Text>
          <Text style={styles.about}>
            TwoCents is a cross pollination student project presented in Module
            4 of Turing School of Software and Design
          </Text>
        </View>
        <View style={styles.yelp}>
          <Text style={styles.yelpText}>Powered by</Text>
          <Image
            source={require("../images/Yelp_trademark_RGB_outline.png")}
            style={{ width: 100, height: 40, marginTop: -15 }}
          />
        </View>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#2C2540',
    alignItems: 'center',
    // paddingTop: 50
  },
  main: {
    height: '90%',
    width: '90%',
    alignItems: 'center',
  },
  text: {
    color: '#EE933F',
    fontSize: 30,
  },
  imageBox: {
    paddingTop: "5%",
    alignItems: "center",
    height: "35%",
    width: "100%"
  },
  about: {
    color: "#CCC0DD",
    fontSize: 24,
    textAlign: "justify",
    paddingTop: 20,
    paddingBottom: 50,
    lineHeight: 0
  },
  yelpText: {
    color: "#CCC0DD",
    fontSize: 16
  },
  yelp: {
    height: '10%',
    width: '100%',
    flexDirection: "row",
    justifyContent: "center",
    alignContent: "center"
  }
});