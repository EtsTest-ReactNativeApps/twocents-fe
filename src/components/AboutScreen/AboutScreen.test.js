import React from 'react';
import { shallow } from 'enzyme';
import { AboutScreen } from './AboutScreen';

describe('AboutScreen', () => {

  it('should match to snapshot', () => {
    const component = shallow(<AboutScreen/>)
    expect(component).toMatchSnapshot()
  });
  
});