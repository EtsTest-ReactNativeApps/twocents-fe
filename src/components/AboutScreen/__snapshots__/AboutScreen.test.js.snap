// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutScreen should match to snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#2C2540",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": "90%",
        "width": "90%",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "height": "35%",
          "paddingTop": "5%",
          "width": "100%",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/images/twocents-logo.png",
          }
        }
        style={
          Object {
            "height": 200,
            "width": 200,
          }
        }
      />
    </View>
    <Text
      style={
        Object {
          "color": "#EE933F",
          "fontSize": 30,
        }
      }
    >
      TwoCents
    </Text>
    <Text
      style={
        Object {
          "color": "#CCC0DD",
          "fontSize": 24,
          "lineHeight": 0,
          "paddingBottom": 50,
          "paddingTop": 20,
          "textAlign": "justify",
        }
      }
    >
      TwoCents is a cross pollination student project presented in Module 4 of Turing School of Software and Design
    </Text>
  </View>
  <View
    style={
      Object {
        "alignContent": "center",
        "flexDirection": "row",
        "height": "10%",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#CCC0DD",
          "fontSize": 16,
        }
      }
    >
      Powered by
    </Text>
    <Image
      source={
        Object {
          "testUri": "../../../src/images/Yelp_trademark_RGB_outline.png",
        }
      }
      style={
        Object {
          "height": 40,
          "marginTop": -15,
          "width": 100,
        }
      }
    />
  </View>
</View>
`;
